rest-api/dao/physical_measurements_dao.py

```diff
@@ -298,8 +298,9 @@ def _update_participant_summary(self, session, obj):
     participant_summary.lastModified = clock.CLOCK.now()
 
     if obj.status and obj.status == PhysicalMeasurementsStatus.CANCELLED:
-      # update corresponding participant summary physicalMeasurementsStatus
-      participant_summary.physicalMeasurementsStatus = obj.status
+      if not self.existing_pm(session, participant):
+        # update corresponding participant summary physicalMeasurementsStatus
+        participant_summary.physicalMeasurementsStatus = obj.status
```

Comment:
opt: more characters, but might be slightly easier to understand if you replaced `obj.status` here with `PhysicalMeasurementsStatus.CANCELLED`

Reply:
done